modules/nf-core/yak/count/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process YAK_COUNT"
+    script "../main.nf"
+    process "YAK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "yak"
+    tag "yak/count"
+
+    test("test_yak_count_methylated_1_fastq_gz") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_methylated_1_fastq_gz'], checkIfExists: true)
+                ]
+                """
+            }
+        }
```

Comment:
I don't think you need the methylated files right, if not don't overcomplicate it. 

```suggestion
    test("sarscov2 - Illumina - se") {

        when {
            process {
                """                
                input[0] = [
                    [ id:'test' ], // meta map
                   file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true),
                ]
                """
            }
        }
```